From 3f51bc789a4a3703a742231a807ec27413e8fecb Mon Sep 17 00:00:00 2001
From: Xiyue Deng <manphiz@gmail.com>
Date: Mon, 27 Jan 2025 23:23:37 -0800
Subject: Fix eldoc warning in patch adding debian-emacs-flavor

Forwarded: not-needed
Closes: #1095133
---
 lisp/startup.el | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lisp/startup.el b/lisp/startup.el
index c6242b39e4f..a17a6d0d959 100644
--- a/lisp/startup.el
+++ b/lisp/startup.el
@@ -443,7 +443,7 @@ tutorial-directory
 
 (defconst debian-emacs-flavor 'emacs
   "A symbol representing the particular debian flavor of emacs running.
-Something like 'emacs, 'xemacs21, etc.")
+Something like \\='emacs, \\='xemacs21, etc.")
 
 (defun normal-top-level-add-subdirs-to-load-path ()
   "Recursively add all subdirectories of `default-directory' to `load-path'.
